iree_tests/pytorch/models/opt-125M/test_data_flags.txt

```diff
@@ -0,0 +1,2 @@
+--input="1x8xi64"
+--parameters=splats.irpa
```

Comment:
A "real weights" version of this test would need
* 2MB of input/output `.bin` files
* the 1.2GB `real_weights.irpa` file
* a flagfile like this:

    ```
    --parameters=real_weights.irpa
    --input=1x8xi64=@inference_input.0.bin
    --expected_output=1x8x50272xf32=@inference_output.0.bin
    --expected_output=1x12x8x64xf32=@inference_output.1.bin
    --expected_output=1x12x8x64xf32=@inference_output.2.bin
    --expected_output=1x12x8x64xf32=@inference_output.3.bin
    --expected_output=1x12x8x64xf32=@inference_output.4.bin
    --expected_output=1x12x8x64xf32=@inference_output.5.bin
    --expected_output=1x12x8x64xf32=@inference_output.6.bin
    --expected_output=1x12x8x64xf32=@inference_output.7.bin
    --expected_output=1x12x8x64xf32=@inference_output.8.bin
    --expected_output=1x12x8x64xf32=@inference_output.9.bin
    --expected_output=1x12x8x64xf32=@inference_output.10.bin
    --expected_output=1x12x8x64xf32=@inference_output.11.bin
    --expected_output=1x12x8x64xf32=@inference_output.12.bin
    --expected_output=1x12x8x64xf32=@inference_output.13.bin
    --expected_output=1x12x8x64xf32=@inference_output.14.bin
    --expected_output=1x12x8x64xf32=@inference_output.15.bin
    --expected_output=1x12x8x64xf32=@inference_output.16.bin
    --expected_output=1x12x8x64xf32=@inference_output.17.bin
    --expected_output=1x12x8x64xf32=@inference_output.18.bin
    --expected_output=1x12x8x64xf32=@inference_output.19.bin
    --expected_output=1x12x8x64xf32=@inference_output.20.bin
    --expected_output=1x12x8x64xf32=@inference_output.21.bin
    --expected_output=1x12x8x64xf32=@inference_output.22.bin
    --expected_output=1x12x8x64xf32=@inference_output.23.bin
    --expected_output=1x12x8x64xf32=@inference_output.24.bin
    ```
    
1.2GB is semi-reasonable for git LFS... mostly dependent on billing and then observed network I/O speeds.

Beyond git LFS, these files could go in a cloud storage bucket, then I'd need to put a reference here (e.g. in a .json file) and teach the test runner and/or CI scripts to fetch those files on-demand.

Reply:
Made progress today:
* Putting files in a cloud bucket
* Adding a download script to pull files from the cloud (or rely on some runner-level cache)
* Teaching conftest.py to read a metadata file and add test cases requiring extra files if those files exist

Need to optimize the download step and then I can push to test on github actions.